src/core/proto/rule_table_mgr.cpp

```diff
@@ -101,71 +101,74 @@ void rule_table_mgr::update_tbl(nl_data_t data_type)
 }
 
 // Parse received rule entry into custom object (rule_val).
-// Parameters:
-//		nl_header	: object that contain rule entry.
-//		p_val		: custom object that contain parsed rule data.
-// return true if its not related to local or default table, false otherwise.
-void rule_table_mgr::parse_entry(struct nlmsghdr *nl_header)
+void rule_table_mgr::parse_entry(struct nl_object *nl_obj)
```

Comment:
add netlink/route/rule.h and netlink/netlink.h includes. See rationale at route_table_mgr

Reply:
done :)